--- darktable/data/darktable.appdata.xml.in	2019-03-20 09:14:56.000000000 +0100
+++ darktable/data/darktable.appdata.xml.in	2019-08-13 19:50:30.715216462 +0200
@@ -51,4 +51,11 @@
   <url type="bugtracker">https://redmine.darktable.org/projects/darktable/issues</url>
   <url type="help">https://www.darktable.org/usermanual/</url>
   <update_contact>darktable-dev@lists.darktable.org</update_contact>
+  <content_rating type="oars-1.0"/>
+  <releases>
+    <release date="2019-10-20" version="2.6.3"/>
+  </releases>
 </component>
