--- darktable/data/darktable.appdata.xml.in	2019-12-21 10:57:37.000000000 +0100
+++ darktable/data/darktable.appdata.xml.in	2019-12-24 15:18:24.562766594 +0100
@@ -49,6 +49,7 @@
   </screenshots>
   <content_rating type="oars-1.1" />
   <releases>
+    <release date="2019-12-24" version="3.0.0"/>
     <release date="2019-03-21" version="2.6.2"/>
     <release date="2019-03-07" version="2.6.1"/>
     <release date="2018-12-24" version="2.6.0"/>
