--- darktable-1/data/org.darktable.darktable.appdata.xml.in-orig	2023-02-23 00:16:50.408327493 -0500
+++ darktable-1/data/org.darktable.darktable.appdata.xml.in	2023-02-23 00:17:03.312545749 -0500
@@ -2,7 +2,7 @@
 <!-- Copyright (C) 2009-2021 darktable developers.
 -->
 <component type="desktop">
-  <id>org.darktable.darktable</id>
+  <id>org.darktable.Darktable</id>
   <name>darktable</name>
   <launchable type="desktop-id">org.darktable.darktable.desktop</launchable>
   <metadata_license>CC0-1.0</metadata_license>
